Neos.ContentRepository.Core/Classes/Feature/NodeCreation/Command/CreateNodeAggregateWithNode.php

```diff
@@ -65,12 +65,11 @@ private function __construct(
      * @param OriginDimensionSpacePoint $originDimensionSpacePoint Origin of the new node in the dimension space. Will also be used to calculate a set of dimension points where the new node will cover from the configured specializations.
      * @param NodeAggregateId $parentNodeAggregateId The id of the node aggregate underneath which the new node is added
      * @param NodeAggregateId|null $succeedingSiblingNodeAggregateId Node aggregate id of the node's succeeding sibling (optional). If not given, the node will be added as the parent's first child
```

Comment:
`$succeedingSiblingNodeAggregateId` might also be worth having a `with*` for to have extra space for documentation :D My guess is that this will not be used directly and the next option `$initialPropertyValues` is super important and having a `null` in between is me :D

Reply:
agreed

Reply:
succeeding sibling is a common requirement – you only don't need it if you want the new node to be the _first_ child (or if you don't care about the position)